tests/integration/test_cubepart.py

```diff
@@ -792,6 +792,22 @@ def it_provides_values_for_cat_with_means_and_insertions(self):
         assert strand.title == "Untitled"
         assert strand.unweighted_counts == (409, 113, 139, 409, 252)
 
+    def it_provides_values_for_univariate_datetime(self):
+        strand = Cube(CR.DATE, population=9001).partitions[0]
```

Comment:
I find the `UNIVARIATE_` prefix redundant. I've left it where I've found it when the fixture is otherwise serviceable just because I didn't want to get side-tracked with too much fixture renovation on this already quite large branch.